src/App.tsx

```diff
@@ -11,11 +11,19 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((prevState) => {
+      return {
+        count: prevState.count + 1,
+      };
+    });
```

Comment:
You can make it short
```suggestion
    this.setState((prevState) => ({
        count: prevState.count + 1,
      });
    );
```